utility_network/configure-subnetwork-trace/src/main/resources/configure_subnetwork_trace/style.css

```diff
@@ -16,33 +17,39 @@ HBox {
   -fx-font-weight: normal;
 }
 
-
 TextArea {
   -fx-min-height: 181;
   -fx-min-width: 630;
 }
 
-ComboBox {
-  -fx-min-width: 175;
-  -fx-max-width: 175;
+.add-button {
+  -fx-min-width: 40;
 }
 
-TextField {
-  -fx-min-width: 175;
+.trace-button {
+  -fx-min-width: 545;
 }
 
-.comparison-choice-element {
+.reset-button {
   -fx-min-width: 175;
 }
 
-.add-button {
-  -fx-min-width: 40;
+.comparison-sources-combobox {
+  -fx-min-width: 240;
+  -fx-max-width: 240;
 }
 
-.trace-button {
-  -fx-min-width: 545;
+.comparison-operators-combobox {
+  -fx-min-width: 240;
+  -fx-max-width: 240;
 }
 
-.reset-button {
+.comparison-choice-elements TextField {
   -fx-min-width: 175;
+  -fx-max-width: 175;
+}
+
+.comparison-choice-elements ComboBox {
+  -fx-min-width: 175;
+  -fx-max-width: 175;
 }
```

Comment:
If there's no other elements other than the one TextField and the one ComboBox inside the styleClass `.comparison-choice-elements `, you might be able to simply do one rule:
```
.comparison-choice-elements {
  -fx-min-width: 175;
  -fx-max-width: 175;
}
```